docs/about/whats-new/whats-new-0.6.0.ipynb

```diff
@@ -0,0 +1,6969 @@
+       "  grid-column: 4;\n",
+       "  text-align: left;\n",
+       "  color: #222;\n",
+       "}\n",
+       ".sc-value-preview {\n",
+       "  grid-column: 5;\n",
+       "}\n",
+       ".sc-var-preview-variances{\n",
+       "  text-align:right;\n",
+       "}\n",
+       "\n",
+       ".sc-sections .sc-section-item .sc-section-summary,\n",
+       ".sc-sections .sc-section-item .sc-section-inline-details,\n",
+       ".sc-section-item .sc-var-list .sc-var-item > div,\n",
+       ".sc-section-item .sc-var-list .sc-var-item > label,\n",
+       ".sc-section-details .sc-var-list .sc-var-item > div,\n",
+       ".sc-section-details .sc-var-list .sc-var-item > label {\n",
+       "  margin-top: 0;\n",
+       "  margin-bottom: 0;\n",
+       "}\n",
+       "\n",
+       ".sc-var-name,\n",
+       ".sc-var-dims,\n",
+       ".sc-var-dtype,\n",
+       ".sc-var-unit,\n",
+       ".sc-preview,\n",
+       ".sc-attrs dt {\n",
+       "  white-space: nowrap;\n",
+       "  overflow: hidden;\n",
+       "  text-overflow: ellipsis;\n",
+       "  padding-right: 10px;\n",
+       "}\n",
+       "\n",
+       ".sc-var-name:hover,\n",
+       ".sc-var-dims:hover,\n",
+       ".sc-var-dtype:hover,\n",
+       ".sc-var-unit:hover,\n",
+       ".sc-attrs dt:hover {\n",
+       "  overflow: visible;\n",
+       "  width: auto;\n",
+       "  z-index: 1;\n",
+       "}\n",
+       "\n",
+       ".sc-var-attrs {\n",
+       "  display: block;\n",
+       "}\n",
+       ".sc-var-data {\n",
+       "  display: none;\n",
+       "}\n",
+       ".sc-var-attrs,\n",
+       ".sc-var-data {\n",
+       "  background-color: #fff !important;\n",
+       "  padding-bottom: 5px !important;\n",
+       "}\n",
+       "\n",
+       ".sc-var-attrs-in:checked ~ .sc-var-attrs {\n",
+       "  display: none;\n",
+       "}\n",
+       ".sc-var-data-in:checked ~ .sc-var-data {\n",
+       "  display: block;\n",
+       "}\n",
+       "\n",
+       ".sc-var-data > table {\n",
+       "  float: right;\n",
+       "}\n",
+       "\n",
+       ".sc-var-name span,\n",
+       ".sc-var-data,\n",
+       ".sc-attrs {\n",
+       "  padding-left: 25px !important;\n",
+       "}\n",
+       "\n",
+       ".sc-attrs,\n",
+       ".sc-var-attrs,\n",
+       ".sc-var-data {\n",
+       "  grid-column: 1 / -1;\n",
+       "}\n",
+       "\n",
+       "dl.sc-attrs {\n",
+       "  padding: 0;\n",
+       "  margin: 0;\n",
+       "  display: grid;\n",
+       "  grid-template-columns: 125px auto;\n",
+       "}\n",
+       "\n",
+       ".sc-attrs dt, dd {\n",
+       "  padding: 0;\n",
+       "  margin: 0;\n",
+       "  float: left;\n",
+       "  padding-right: 10px;\n",
+       "  width: auto;\n",
+       "}\n",
+       "\n",
+       ".sc-attrs dt {\n",
+       "  font-weight: normal;\n",
+       "  grid-column: 1;\n",
+       "}\n",
+       "\n",
+       ".sc-attrs dt:hover span {\n",
+       "  display: inline-block;\n",
+       "  background: #fff;\n",
+   
```

Comment:
Combine this line with the one below? i.e.

from_mantid (and by extension load) have been improved to support loading of McStasNexus files and WorkspaceGroup workspaces.

Reply:
👍